src/configure.md

```diff
@@ -293,6 +294,14 @@ and `V` to `<C-W>m` in Normal and Visual mode:
 "V" = "<C-W>m"
 ```
 
+You can also use this to trigger commands. For example, to list all chats 
+when you press 'gd' in normal mode use:
```

Comment:
I think you meant this to be `'gc'` like in the example below.

Reply:
ahh typo's, my greatest nemesis. Thanks for catching this ill fix it :+1: